### ISSUE TITLE
"Wipe free space" does not work properly

### ISSUE THREAD

**Issue by Haurabe** [bleachbit/bleachbit#1174]
When you use the "wipe free space" function at the very end, the program just freezes up and you have the option to either wait or close it. (but waiting did not do anything)
On the disk itself appears a file that takes all the free space and have to delete it manually. (And this clearly should not be)
P.S.
The program installed on the system disk (where I try to wipe the free space)
Version - bleachbit 4.2.0
PC - Win10 x64 20H2 19042.685

**Comment by az0** [comment]
How long did you wait? Do you have a HDD or SSD? How much space is free on it (when not cleaning it)?

So far, this seems like a side effect of 

1. The cleaning work is done in the same process as the user interface, so in certain situations the UI becomes non-responsive
2. At the end of cleaning, deleting a single large file can be slow because of how the operating system works

In the future, the work will be done in a separate process, to the UI is more responsive. Also, the UI can give more feedback on what is happening.

**Comment by Haurabe** [comment]
Yes, I performed this operation on an SSD drive.(There was about 160 gb of free space out of 223 Gb before I started doing the free space cleanup)
It seems like it should have already ended, but the program interface just hung up and that's it.
I tried to wait for about half an hour but nothing changed.
The disk itself is a system disk.
Only shutting down the process via the task manager and manually deleting a file weighing 160Gb helped.
